src/Modules/UngradedModules.jl

```diff
@@ -1234,6 +1530,122 @@ function cokernel(A::MatElem)
   return cokernel(map(A))
 end
 
+@doc Markdown.doc"""
+    default_ordering(M::SubQuo)    
+
+Return the default ordering of `M`.
+"""
+function default_ordering(M::SubQuo)
+  if !isdefined(M.sub, :default_ordering)
+    try
+      ord = default_ordering(ambient_free_module(M))
+      set_default_ordering!(M, ord)
+      return default_ordering(M.sub)
+    catch _
+      error("The module has no default ordering")
+    end
+  end
+  return default_ordering(M.sub)
```

Comment:
Same question as before: when would an error be thrown that has to be handled here? And if it happens, why do we need to "handle" it by throwing an exception of our own, instead of just letting the original exception through?
```suggestion
  if !isdefined(M.sub, :default_ordering)
    ord = default_ordering(ambient_free_module(M))
    set_default_ordering!(M, ord)
  end
  return default_ordering(M.sub)
```